### PR TITLE
Fixes some camera errors, fighter docking slightly cleaned up

### PR DESCRIPTION
**Pull request by Kmc2000**
## Changelog
:cl:
fix: Resolved some bugs with Galactic Conquest
/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/fighters/fighters_launcher.dm [hunk 1]
```diff
@@ -289,17 +289,16 @@
 	last_overmap = get_overmap()
 
 /obj/structure/overmap/fighter/proc/docking_act(obj/structure/overmap/OM)
-	var/obj/item/fighter_component/docking_computer/DC = loadout.get_slot(HARDPOINT_SLOT_DOCKING)
-	if(mass < OM.mass && OM.docking_points.len && DC?.docking_mode) //If theyre smaller than us,and we have docking points, and they want to dock
+	if(mass < OM.mass) //If theyre smaller than us,and we have docking points, and they want to dock
```
Comment (ondrej008) [inline nsv13/code/modules/overmap/fighters/fighters_launcher.dm:292]: sir where did the docking_mode go

Comment (Kmc2000) [reply nsv13/code/modules/overmap/fighters/fighters_launcher.dm:292]: Moved down to transfer_from_overmap

Comment (Kmc2000) [reply nsv13/code/modules/overmap/fighters/fighters_launcher.dm:292]: Good point, didn't see that

File: nsv13/code/modules/overmap/fighters/fighters_launcher.dm [hunk 2]
```diff
@@ -289,17 +289,16 @@
...
 		return transfer_from_overmap(OM)
 	else
 		return FALSE
 
 /obj/structure/overmap/fighter/proc/transfer_from_overmap(obj/structure/overmap/OM)
 	var/obj/item/fighter_component/docking_computer/DC = loadout.get_slot(HARDPOINT_SLOT_DOCKING)
-	if(!DC || DC.docking_cooldown)
+	if(!DC || DC.docking_cooldown || !OM.occupying_levels?.len)
```
Comment (Kmc2000) [inline nsv13/code/modules/overmap/fighters/fighters_launcher.dm:299]:
```suggestion
	if(!DC || DC.docking_cooldown ||!DC.docking_mode|| !OM.occupying_levels?.len)
```

**Pull request merged**